rules/integrations/aws/defense_evasion_cloudtrail_logging_suspended.toml

```diff
@@ -24,7 +24,64 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS CloudTrail Log Suspended"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating AWS CloudTrail Log Suspended
+
+Amazon CloudTrail is a service that enables governance, compliance, operational auditing, and risk auditing of your
+Amazon Web Services account. With CloudTrail, you can log, continuously monitor, and retain account activity related to
+actions across your Amazon Web Services infrastructure. CloudTrail provides event history of your Amazon Web Services
+account activity, including actions taken through the Amazon Management Console, Amazon SDKs, command line tools, and
+other Amazon Web Services services. This event history simplifies security analysis, resource change tracking, and
+troubleshooting.
+
+This rule identifies the deletion of an AWS log trail using the API `DeleteTrail` action. Attackers can do this to
+cover their tracks and impact security monitoring that relies on this source.
```

Comment:
All of the content for this rule is exactly the same as "Investigating AWS CloudTrail Log Deleted" above. Is there any difference between deleting and suspending a CloudTrail log? For example, should this paragraph say something like "the _suspension_ of an AWS log trail", and is there a different action for suspending (guessing `SuspendTrail` or something)?

Reply:
Good catch, i'll fix the description, but for investigation steps, they are basically the same

Reply:
Done